bioimageio/spec/delegator.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import annotations
```

Comment:
why is this file called delegator?

Reply:
it delegates the io function to the respective submodule

Reply:
of course 'utils' or similar would also be appropriate, but there are already many files called 'utils.py' in here...

Reply:
Is this imported in the top-level, or do library users need to do `from spec.delegator import something`?